addons/ui/flexiMenu/init.sqf

```diff
@@ -8,22 +8,6 @@ GVAR(target) = objNull; // current object for interaction
 GVAR(holdKeyDown) = false; // default global behaviour of requiring holding key down. Can be overriden by menus.
 GVAR(hotKeyColor) = ""; // override colour of hotkey. It is set from rsc menu file, upon opening either the "menu" or "list". If not present, a default will be used.
 
-PREP_SUB(flexiMenu,keyDown);
-PREP_SUB(flexiMenu,keyUp);
-PREP_SUB(flexiMenu,menu);
-PREP_SUB(flexiMenu,list);
-PREP_SUB(flexiMenu,getMenuDef);
-PREP_SUB(flexiMenu,getMenuOption);
-PREP_SUB(flexiMenu,menuShortcut);
-PREP_SUB(flexiMenu,mouseButtonDown);
-PREP_SUB(flexiMenu,highlightCaretKey);
-PREP_SUB(flexiMenu,execute);
-
-// TODO: move these to public CfgFunctions soon.
-PREP_SUB(flexiMenu,add);
-PREP_SUB(flexiMenu,remove);
-PREP_SUB(flexiMenu,setObjectMenuSource);
-
```

Comment:
```suggestion
FUNC(keyDown) = CBA_fnc_flexiMenu_keyDown;
FUNC(keyUp) = CBA_fnc_flexiMenu_keyUp;
FUNC(menu) = CBA_fnc_flexiMenu_menu;
FUNC(list) = CBA_fnc_flexiMenu_list;
FUNC(getMenuDef) = CBA_fnc_flexiMenu_getMenuDef;
FUNC(getMenuOption) = CBA_fnc_flexiMenu_getMenuOption;
FUNC(menuShortcut) = CBA_fnc_flexiMenu_menuShortcut;
FUNC(mouseButtonDown) = CBA_fnc_flexiMenu_mouseButtonDown;
FUNC(highlightCaretKey) = CBA_fnc_flexiMenu_highlightCaretKey;
FUNC(execute) = CBA_fnc_flexiMenu_execute;
FUNC(add) = CBA_fnc_flexiMenu_add;
FUNC(remove) = CBA_fnc_flexiMenu_remove;
FUNC(setObjectMenuSource) = CBA_fnc_setObjectMenuSource;
```

tbh, this whole subcomponent is cancer.

Reply:
Are you sure about this? could just move the old PREP_SUB that was there to PREP.hpp and get same effect as on master branch

Reply:
PREP_SUB is there because the filepath is different. The files are in a sub folder.

Reply:
I know. But does anything speak against just using the old PREP_SUB code in PREP.hpp? Instead of just copying the functions to a new/old name?

Reply:
Idk how PREP_SUB works. I just assume it recompiles every time and is nothing like PREP at all.

Reply:
Correct. Direct compile preproc..